core/src/main/scala/org/apache/spark/scheduler/cluster/mesos/CoarseMesosSchedulerBackend.scala

```diff
@@ -20,16 +20,16 @@ package org.apache.spark.scheduler.cluster.mesos
 import java.io.File
 import java.util.{Collections, List => JList}
 
-import scala.collection.JavaConversions._
-import scala.collection.mutable.{HashMap, HashSet}
-
 import org.apache.mesos.Protos.{TaskInfo => MesosTaskInfo, _}
 import org.apache.mesos.{Scheduler => MScheduler, _}
 import org.apache.spark.scheduler.TaskSchedulerImpl
 import org.apache.spark.scheduler.cluster.CoarseGrainedSchedulerBackend
+import org.apache.spark.scheduler.cluster.mesos.MesosUtils._
 import org.apache.spark.util.{AkkaUtils, Utils}
 import org.apache.spark.{SparkContext, SparkEnv, SparkException, TaskState}
 
+import scala.collection.mutable.{HashMap, HashSet}
```

Comment:
scala imports needs to be after java imports according to spark style guide, please move in between java and org.apache imports.